code/modules/NTNet/netdata.dm

```diff
@@ -7,6 +7,22 @@
 	var/plaintext_data
 	var/plaintext_data_secondary
 	var/plaintext_passkey
```

Comment:
should just rename this to passkey_encrypted, and start including a list of what can be packet-sniffed and what can't be packet-sniffed.

Reply:
good call